lib/tmuxinator/config.rb

```diff
@@ -56,6 +60,15 @@ def default
         "#{directory}/default.yml"
       end
 
+      def options
+        yaml = begin
+          YAML.safe_load(File.read(options_file), [], [], true)
+        rescue SyntaxError, StandardError
+          return {}
+        end
+        yaml ? yaml : {}
+      end
```

Comment:
I believe this is equivalent:

```suggestion
      def options
        YAML.safe_load(File.read(options_file), [], [], true) || {}
      rescue SyntaxError, StandardError
        {}
      end
```